cost/google/idle_ip_address_recommendations/google_idle_ip_address_recommendations.pt

```diff
@@ -392,13 +392,13 @@ policy "policy_recommendations" do
     export "instances" do
       resource_level true
       field "accountID" do
-        label "Account ID"
-      end
-      field "accountNumber" do
-        label "Account Number"
+        label "Project ID"
       end
       field "accountName" do
-        label "Account Name"
+        label "Project Name"
+      end
+      field "projectNumber" do
```

Comment:
cannot find a field called `projectNumber`, can only see `accountNumber `referenced previously in the template